### DISCUSSION TITLE
✅ Slider widget custom design per slide

### DISCUSSION THREAD

**Discussion by doumlegare** [elementor/elementor#27713] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

The ability to have more control on the slider widget with a custom design

### Describe the solution you'd like.

Implement container per slide to have custom design for each slide of the slider

### Describe alternatives you've considered.

3rd party plugins it’s the only alternative

### Additional context

The goal is to implement the container module per slide  like you did for toggle/ accordion/ off canvas widget.
I think is simple not to complicated to implement for you team because you already did it for multiple widget.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by rami-elementor** [comment]
@doumlegare  You can do that with Elementor.

Activate the "Containers" and the "Nested Elements" features.

Select the "Carousel" widget and add as many slide you need. Now drag to each slide. you can choose different widgets to different slides.
